### PR TITLE
K8SPSMDB-764 fix operator image and delete unused mongo versions

### PR DESCRIPTION
**Pull request by nmarukovich**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.13.0.psmdb-operator.json [hunk 1]
```diff
@@ -53,89 +47,11 @@
...
-            "image_hash": "991d6049059e5eb1a74981290d829a5fb4ab0554993748fde1e67b2f46f26bf0",
-            "status": "available",
-            "critical": false
-          },
           "4.2.22-22": {
             "image_path": "percona/percona-server-mongodb:4.2.22-22",
             "image_hash": "sha256:da4634df780563e10a547662c58c8ce28fbe5c98e1ac8b42b4f6be87f292e92b",
             "status": "recommended",
             "critical": false
-          },
```
Comment (tplavcic) [inline sources/operator.1.13.0.psmdb-operator.json:85]: Why are we removing all except one 4.2 version?

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | failed |

File: sources/operator.1.13.0.psmdb-operator.json (hunk 1 above)
Comment (nmarukovich) [reply sources/operator.1.13.0.psmdb-operator.json:85]: version 1.13.0 works only with 4.2.22-22
cc @hors

Comment (hors) [reply sources/operator.1.13.0.psmdb-operator.json:85]: @tplavcic we have added [js-yaml](https://github.com/percona/percona-docker/commit/c18c1737d1def5d78524f7e740478dd8a8ecf66e) to the latest psmdb 4.2 image. Without this tool operator can't work correctly  with psmdb image (starting from 1.13).

Comment (nmarukovich) [reply sources/operator.1.13.0.psmdb-operator.json:85]: I also delete all 4.4 and 5.0 version which don't have this plugin.

**Review by tplavcic — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**